packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -269,7 +269,19 @@ Blockly.BlockSvg.prototype.setCollapsed = function (collapsed) {
             this.setCollapsed(false)
         );
 
-        this.appendDummyInput(COLLAPSED_INPUT_NAME).appendField(field_label).appendField(field_expand_icon).init();
+        if (this.type === 'procedures_defreturn' || this.type === 'procedures_defnoreturn') {
+            const funcLocale = this.getInput('').fieldRow[0].text_;
+            const funcName = this.getFieldValue('NAME');
+            const args = ` (${this.arguments.join(', ')})`;
+
+            this.appendDummyInput(COLLAPSED_INPUT_NAME)
+                .appendField(new Blockly.FieldLabel(funcLocale, field_class))
+                .appendField(new Blockly.FieldLabel(funcName + args, 'header__title'))
+                .appendField(field_expand_icon)
+                .init();
+        } else {
+            this.appendDummyInput(COLLAPSED_INPUT_NAME).appendField(field_label).appendField(field_expand_icon).init();
```

Comment:
As if statement is not related to some const such as text is it possible to put the content which related to else statement inside it to make it clear